src/markdown-pages/docs/running-simbank-tests/web-app-integration-test.md

```diff
@@ -0,0 +1,162 @@
+
+The `WebAppIntetgrationTest` test demonstrates the end-to-end integration testing of an application that runs on different platforms (z/OS and Cloud) and uses different technologies (a 3270 emulator, JCL batch job and Selenium Web Driver).
+
+The following diagram shows how you can use the WebAppIntegrationTest to test a hybrid cloud application: 
+
+![Galasa in hybrid cloud](../../galasa-hybrid-cloud.svg)
+
+The diagram highlights the use of the following technology, and the interaction between the test and those technologies.
+
+ - The 3270 emulator creates a random account number and checks that it does not exist in the CICS region
+ - If the account does not exist the account is created by the Z/OS Batch Manager and is inserted into the backend database
+ - Docker then generates a web page
+ -  Selenium drives the web browser to complete a web form and confirm the balance of the account
+
+
+## Running the WebAppIntegrationTest
+
+As the `WebAppIntetgrationTest` is slightly different to the other tests, there's a bit more set up to do than for the other supplied SimBank tests. 
+
+The test is still run locally but is designed to help you to understand how to set the properties to enable the test to run in a Galasa Ecosystem. When running locally, you can use the localhost setting `127.0.0.1` for increased security. 
+
+The test uses the Selenium Manager, which in turn is dependent on the Docker Manager. Use the following sections to help you to understand how to configure your environment to work with the Selenium and Docker Manager. 
+
+### Using the Selenium Manager
+
+The `WebAppIntetgrationTest` uses the Selenium Manager. To use the Selenium Manager you must have a web browser, for example, Chrome or Firefox installed as well as either a gecko driver or a docker engine. In this example, Firefox and a gecko driver are used. 
+
+You can <a href="https://github.com/mozilla/geckodriver/releases" target="_blank"> download Gecko driver from GitHub</a>. 
+
+You must define the default and local driver properties in the CPS, as shown in the following example:
+
+```
+selenium.default.driver=FIREFOX
+selenium.local.driver.FIREFOX.path=<path/to/geckodriver>
+```
+
+### Using the Docker Manager 
+
+The Selenium Manager has a dependency on the Docker Manager in order to run. Some set up is required for the Docker Manager in the CPS properties file. To configure the Docker Manager, set the following CPS properties:  
+
+```
+docker.dse.engine.PRIMARY=LOCAL
+docker.default.engines=LOCAL
+docker.engine.LOCAL.hostname=127.0.0.1
+docker.engine.local.port=2375
+docker.engine.local.max.slots=10
```

Comment:
some inconsistency in the capitalisation of local

Reply:
capitalised, thank you

Reply:
cool thanks